pyart/retrieve/qvp.py

```diff
@@ -25,6 +25,9 @@ def quasi_vertical_profile(radar, desired_angle=None, fields=None, gatefilter=No
     desired_angle : float
         Radar tilt angle to use for indexing radar field data.
         None will result in wanted_angle = 20.0
+    verbose : bool
+        boolean flag to turn the printing of radar tilts on or off. 
+        Can be very annoying if you process alot of files. 
```

Comment:
Yeah - I would agree with @zssherman here, can you remove this line? The rest looks great 😁

Reply:
Git novice over here. How do I make a change to an exisitng pull request?

Reply:
or to your main

Reply:
You'll want to make this change on your branch, commit, then push again